prow/config/jobs/common-files-1.10.yaml

```diff
@@ -0,0 +1,172 @@
+branches:
+- release-1.10
+image: gcr.io/istio-testing/build-tools:release-1.10-2021-04-02T23-15-48
+jobs:
+- command:
+  - make
+  - lint
+  name: lint
+  node_selector:
+    testing: test-pool
+  requirements:
+  - cache
+- command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=istio,api,tools,release-builder,pkg,client-go,gogo-genproto,proxy
+  - '--title=Automator: update common-files@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH'
+  - --labels=auto-merge,release-notes-none
+  - --strict
+  - --modifier=commonfiles
+  - --token-path=/etc/github-token/oauth
+  - --cmd=make update-common && make gen
+  name: update-common
+  node_selector:
+    testing: test-pool
+  repos:
+  - istio/test-infra@master
+  requirements:
+  - cache
+  - github
+  types:
+  - postsubmit
+- command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=istio.io
+  - '--title=Automator: update common-files@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH'
+  - --labels=auto-merge,release-notes-none
+  - --strict
+  - --modifier=commonfiles
+  - --token-path=/etc/github-token/oauth
+  - --cmd=make update-common && make gen
+  name: update-common-istio.io
+  node_selector:
+    testing: test-pool
+  repos:
+  - istio/test-infra@master
+  requirements:
+  - cache
+  - github
+  types:
+  - postsubmit
+- command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=test-infra
+  - '--title=Automator: update build-tools:$AUTOMATOR_SRC_BRANCH'
+  - --branch=master
```

Comment:
is this right?

Reply:
Need to investigate. It is `master` in the other release yamls.

Reply:
Reason this is master is that the repo is test-infra and that only has `master`.